spec/system/scoreboard_spec.rb

```diff
@@ -40,6 +40,7 @@
       visit scoreboard_path(scoreboard)
       expect(page).to have_content('Super scoreboard')
       expect(page).to have_content(school_with_points.name)
+      expect(page).to have_link(points.to_s, href: school_timeline_path(school_with_points, academic_year: scoreboard.this_academic_year))
```

Comment:
to_s not needed?

Reply:
Without it, I get the following warning: 

`Integer:123 for selector :link must be an instance of String or Symbol.`